sdk/storage/azure-storage-blob/apilisting.txt

```diff
@@ -0,0 +1,591 @@
+class AccessPolicy(Model, object):
+	permission:permission
+	expiry:expiry
+	start:start
+
+	def __eq__(self, other): ...
+	def __init__(self, permission: permission=None, expiry: expiry=None, start: start=None): ...
```

Comment:
@rakshith91, what is the semantics if you create an empty AccessPolicy (i.e. don't specify any parameters to the constructor)

Reply:
Default permission is None, same with start and expiry

Reply:
I believe permission shouldn't be optional